### PR TITLE
Remove pandas as a dependency for frame.py.

### PR DESCRIPTION
**Pull request by wendycwong**

### CODE REVIEW

File: h2o-py/h2o/estimators/estimator_base.py [hunk 1]
```diff
@@ -412,6 +412,9 @@ def _metrics_class(model_json):
 
     def convert_H2OXGBoostParams_2_XGBoostParams(self):
         '''
+        This method and convert_H2OFrame_2_DMatrix will require to import the following toolboxes:
+        xgboost, pandas, numpy and scipy.sparse.
+
```
Comment (angela0xdata) [inline h2o-py/h2o/estimators/estimator_base.py]: In order to use convert_H2OXGBoostParams_2_XGBoostParams and convert_H2OFrame_2_DMatrix, you must import the following toolboxes:

File: h2o-py/h2o/frame.py [hunk 2]
```diff
@@ -2568,6 +2567,9 @@ def isax(self, num_words, max_cardinality, optimize_card=False, **kwargs):
 
     def convert_H2OFrame_2_DMatrix(self, predictors, yresp, h2oXGBoostModel):
         '''
+        This method will require to import the following toolboxes:
+        xgboost, pandas, numpy and scipy.sparse.
+
```
Comment (angela0xdata) [inline h2o-py/h2o/frame.py]: This method requires that you import the following toolboxes:

File: h2o-py/h2o/estimators/estimator_base.py (hunk 1 above)
Comment (wendycwong) [reply h2o-py/h2o/estimators/estimator_base.py]: Thanks.  Done

File: h2o-py/h2o/frame.py (hunk 2 above)
Comment (wendycwong) [reply h2o-py/h2o/frame.py]: Done

**Comment by michalkurka** [comment]
@wendycwong looks good to me!

Can you please also make fix on `rel-xia`?

**Comment by wendycwong** [comment]
michalk:  Will add it to rel-xia

**Pull request merged**